lib/membrane/rtp/utils.ex

```diff
@@ -17,6 +17,21 @@ defmodule Membrane.RTP.Utils do
     end
   end
 
+  @doc """
+  Generates padding binary.
+
+  `padding_size` is the final padding size including
+  the last byte denoting padding size. `padding_size`
+  cannot be greater than 255.
+  """
+  @spec generate_padding(non_neg_integer()) :: binary()
```

Comment:
Let's include padding size constraints in the spec
```suggestion
  @spec generate_padding(0..255) :: binary()
```

Reply:
I'd define a type `padding_size` somewhere and reference it here as it is also used in the spec of `Packet.serialize` and the result type of `Packet.parse`